tools/acrn-manager/acrnd.service

```diff
@@ -1,9 +1,15 @@
 [Unit]
 Description=ACRN manager deamon
+After=acrnprobe.service
+After=weston.service
+After=systemd-resolved.service
```

Comment:
why is there dependency on "Network Name Resolution" service?

Reply:
I think it's better to keep this line, because UOS launch script will set up network on SOS...